log_drains/list_addon.go

```diff
@@ -0,0 +1,83 @@
+package log_drains
+
+import (
+	"os"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/go-scalingo"
+	"github.com/olekukonko/tablewriter"
+	"gopkg.in/errgo.v1"
+)
+
+type addonObject struct {
+	Drains    []scalingo.LogDrain
+	AddonName string
+}
+
+func ListAddon(app string, addonID string) error {
```

Comment:
The more I think about it, the more I think you should not add a new function but use the `List` you already wrote. There is a lot of code in common! What do you think?

Reply:
I was afraid it was gonna turn into a big if else, in addition of the condition of with-addon / with id.
But we do find the list code into the list_addon code and we have also duplication in the `cmd/log_drains.go` file too.